File: Iniciante/Java/Beecrowd1009.java
import java.util.Scanner;

public class Beecrowd1009 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        double salario, vendas,  bonus;

        salario = scanner.nextDouble();
        vendas = scanner.nextDouble();

        bonus = vendas*0.15;

        System.out.printf("TOTAL = R$ %.2f\n", salario+bonus);

        scanner.close();

    }
}

File: Iniciante/C/beecrowd1009.c
#include <stdio.h>

int main(void){

    char nome[10];
    double salario, vendas, bonus, total;

    scanf("%s",&nome);
    scanf("%lf",&salario);
    scanf("%lf",&vendas);

    bonus = vendas * 0.15;
    total = salario+bonus;
    printf("TOTAL = R$ %.2lf\n", total);

    return 0;
}

